gamestonk_terminal/etf/screener_model.py

```diff
@@ -1,201 +1,37 @@
-                    query = f"{col} < {cf[col]['Max']} "
-                    df = df.query(query)
-            param_string += "_from_config"
-        else:
-
-            if ns_parser.min_price:
-                df = df.query(f"Price > {ns_parser.min_price}")
-                param_string += f"_p_{ns_parser.min_price}".replace(".", "p")
-
-            if ns_parser.max_price:
-                df = df.query(f"Price < {ns_parser.max_price}")
-                param_string += f"_P_{ns_parser.max_price}".replace(".", "p")
 
-            if ns_parser.min_assets:
-                df = df.query(f"Assets > {ns_parser.min_assets}")
-                param_string += f"_a_{ns_parser.min_assets}".replace(".", "p")
-
-            if ns_parser.max_assets:
-                df = df.query(f"Assets < {ns_parser.max_assets}")
-                param_string += f"_A_{ns_parser.max_assets}".replace(".", "p")
-
-            if ns_parser.min_nav:
-                df = df.query(f"NAV > {ns_parser.min_nav}")
-                param_string += f"_n_{ns_parser.min_nav}".replace(".", "p")
-
-            if ns_parser.max_nav:
-                df = df.query(f"NAV < {ns_parser.max_nav}")
-                param_string += f"_N_{ns_parser.max_nav}".replace(".", "p")
-
-            if ns_parser.min_exp:
-                df = df.query(f"Expense > {ns_parser.min_exp}")
-                param_string += f"_e_{ns_parser.min_exp}".replace(".", "p")
-
-            if ns_parser.max_exp:
-                df = df.query(f"Expense < {ns_parser.max_exp}")
-                param_string += f"_E_{ns_parser.max_exp}".replace(".", "p")
-
-            if ns_parser.min_pe:
-                df = df.query(f"PE > {ns_parser.min_pe}")
-                param_string += f"_r_{ns_parser.min_pe}".replace(".", "p")
-            if ns_parser.max_pe:
-                df = df.query(f"PE < {ns_parser.max_pe}")
-                param_string += f"_R_{ns_parser.max_pe}".replace(".", "p")
-
-            if ns_parser.min_div:
-                df = df.query(f"DivYield > {ns_parser.min_div}")
-                param_string += f"_d_{ns_parser.min_div}".replace(".", "p")
-            if ns_parser.max_div:
-                df = df.query(f"DivYield < {ns_parser.max_div}")
-                param_string += f"_D_{ns_parser.max_div}".replace(".", "p")
-
-            if ns_parser.min_beta:
-                df = df.query(f"Beta > {ns_parser.min_beta}")
-                param_string += f"_b_{ns_parser.min_beta}".replace(".", "p").replace(
-                    "-", "neg"
-                )
-
-            if ns_parser.max_beta:
-                df = df.query(f"Beta < {ns_parser.max_beta}")
-                param_string += f"_B_{ns_parser.max_beta}".replace(".", "p").replace(
-                    "-", "neg"
-                )
+    df = pd.read_csv(
+        "https://raw.githubusercontent.com/jmaslek/etf_scraper/main/etf_overviews.csv",
+        index_col=0,
+    )
+    print("ETFs downloaded\n")
```

Comment:
I don't think this print is needed

Reply:
Mostly as a check that my GitHub is fine.  Ill delete it

Reply:
I think if something fails is better to say "problem with github" but if we get the ETF we are assuming that the github was fine :p